mantidproject/mantid
# Indirect - ConvFit FABADA FixTie is not working

**Description of work.**
This PR solves a problem where a fix (tie) on a parameter when using the FABADA minimizer does not remain fixed.

**To test:**
1. `Interfaces`->`Indirect`->`Data Analysis` -> `ConvFit` Tab
2. Load in the files below
3. In the Function browser:
- `One Lorentzian` for Fit Type
- `Delta Function` should be ticked
- `FlatBackground` as the background
- `Tie` Delta.Centre to Lor.PeakCentre (this tests tie)
- `Fix` FlatBgd.A0=0, it appears as Tie=0 (this tests Fix)
- `Constrain` both bounds for Delta.Height 0.0 to 0.01 (this tests constrain)
- `FABADA` for minimizer
- `Max Iterations` should be around 4000
4. Set `Fit Spectra` string to 0 in order to speed up the process
5. Click `Run` and wait a while
6. The FlatBgd.A0 Tie should remain fixed at zero!

**Files**
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2424131/irs26173_graphite002_res.zip)
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2424132/irs26176_graphite002_red.zip)

Fixes #23544 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
